ci-operator/templates/cluster-launch-e2e-azure.yaml

```diff
@@ -195,11 +199,15 @@ objects:
           export HOME=/tmp/shared
           export DNS_DOMAIN=osadev.cloud
           export DNS_RESOURCEGROUP=dns
+          export KUBECONFIG=/tmp/shared/_data/_out/admin.kubeconfig
   
           cp -r /tmp/shared/_data /go/src/github.com/openshift/openshift-azure/
           cd /go/src/github.com/openshift/openshift-azure/
           source /etc/azure/credentials/secret
           az login --service-principal -u ${AZURE_CLIENT_ID} -p ${AZURE_CLIENT_SECRET} --tenant ${AZURE_TENANT_ID} &>/dev/null
+          oc get po --all-namespaces -o wide > /tmp/artifacts/pods
+          oc get no -o wide > /tmp/artifacts/nodes
+          oc get events --all-namespaces > /tmp/artifacts/events
```

Comment:
These will be gathered even when the container exists successfully.

It might be what you want, but if you need it just for failures, send it to stdout/err.

Otherwise LGTM.

Reply:
Gathering on success is fine